client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -188,7 +193,7 @@ const SiteMigrationInstructions: Step = function ( { navigation, flow } ) {
 		</div>
 	);
 
-	const questions = <Questions />;
+	const supportNudge = <SupportNudge navigateToDoItForMe={ navigateToDoItForMe } />;
```

Comment:
We can remove this intermediated const

Reply:
Handled in 48f56747a928b7612d1529a55fa742e80afc06ca.